app/util/planParamUtil.js

```diff
@@ -186,6 +202,13 @@ export function planQueryNeeded(
         settings.includeBikeSuggestions
       );
 
+    case PLANTYPE.SCOOTERTRANSIT:
+      return (
+        transitModes.length > 0 &&
+        !wheelchair &&
+        settings.allowedScooterRentalNetworks > 0
+      );
+
```

Comment:
Is it by design that plain scooter routes are not searched?

Reply:
Yes, scooters are only supposed to be shown together with some type of public transportation.